ubuntu-deploy/index.md

```diff
@@ -0,0 +1,414 @@
+---
+title: 'Hosting a Directus Application on an Ubuntu Server'
+description: 'Learn how to host a Directus application on Ubuntu server with Docker, Nginx, and SSL encryption. A step-by-step guide for seamless deployment and security.'
+author:
+  name: 'Yusuf Akorede'
+  avatar_file_name: 'avatar.png'
+---
+
+## Introduction
+
+In this tutorial, you will learn how to deploy a Directus application within a Docker container on an Ubuntu Linux server and connect it to a custom domain, such as `https://directus.exampledomain.com`.
+
+This guide covers setting up Docker, configuring Docker Compose, using Nginx as a reverse proxy, and securing your application with SSL. Additionally, you'll discover how to run the application as a background service, ensuring seamless operation and easy management.
+
+## Prerequisites
+
+1. **A Directus Project:** Prepare a local Directus project for deployment. Follow the [Directus quickstart guide][quickstart] if you need to create a new project.
+
+2. **Ubuntu Server (Version 20.04 or 22.04):** Access an Ubuntu server via SSH. Obtain one from cloud providers like Azure, DigitalOcean, Linode, or AWS. Configure SSH access from your local machine.
+
+3. **Domain Name:** Register a domain name and have access to its DNS settings. This is crucial for the domain to resolve to your Ubuntu server
+
+4. **Basic Command-Line Skills:** Basic familiarity with Linux command-line operations, including file uploads using `scp` and editing files with `vi`.
+
+## Upload Your Local Directus Application Folder to the Server
+
+Assuming you have a local Directus folder structure that looks like this:
+
+```plaintext
+├── database
+│   └── data.db
+├── docker-compose.yml
+├── uploads
+└── extensions
+```
+
+And the _docker-compose.yml_ file looks somewhat like this:
+
+```yaml
+version: '3'
+services:
+  directus:
+    image: directus/directus:latest
+    ports:
+      - 8055:8055
+    volumes:
+      - ./database:/directus/database
+      - ./uploads:/directus/uploads
+      - ./extensions:/directus/extensions
+    environment:
+      KEY: 'replace-with-random-value'
+      SECRET: 'replace-with-random-value'
+      ADMIN_EMAIL: 'admin@example.com'
+      ADMIN_PASSWORD: 'd1r3ctu5'
+      DB_CLIENT: 'sqlite3'
+      DB_FILENAME: '/directus/database/data.db'
+      WEBSOCKETS_ENABLED: 1
+      # Add your other settings
+```
+
+:::info Security Note
+
+Ensure you adjust the email and set a strong password for the admin user.
+
+:::
```

Comment:
Because this is exactly the same as the quickstart guide, and we don't later edit the `docker-compose.yml` file, we can link our and summarize here. Something like: 

> If you have successfully followed the [Self-Hosted Quickstart][quickstart], you should have a directory with a `docker-compose.yml` file, `database/` directory with a `data.db` file, `uploads/` directory, and `extensions/` directory.